### PR TITLE
Add example for alias for fs fixture

### PR DESCRIPTION
**Pull request by mrbean-bremen**
- see #609

As proposed by @jmcgeheeiv.

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -50,6 +50,16 @@ plugins, so you can just use it:
        fs.create_file('/var/data/xx1.txt')
        assert os.path.exists('/var/data/xx1.txt')
 
+If you don't like the ``fs`` fixture name (``pylint`` may complain about the
```
Comment (jmcgeheeiv) [inline docs/usage.rst:53]: If you are bothered by PyLint warning,
`C0103: Argument name "fs" doesn't conform to snake_case naming style (invalid-name)`,
you can define a longer name in your ``conftest.py`` and use that in your tests:

File: docs/usage.rst [hunk 2]
```diff
@@ -50,6 +50,16 @@ plugins, so you can just use it:
...
+name), you may define your own alias in your ``conftest.py``:
+
+.. code:: python
+
+    @pytest.fixture
+    def fake_filesystem(fs):
```
Comment (jmcgeheeiv) [inline docs/usage.rst:59]: No `# pylint:disable=invalid-name`?

Comment (jmcgeheeiv) [reply docs/usage.rst:59]: """Variable name 'fs' causes a PyLint warning. Provide a longer name acceptable
to PyLint for use in tests.
"""

Comment (mrbean-bremen) [reply docs/usage.rst:59]: Thanks! Will make the changes tomorrow.

Comment (mrbean-bremen) [reply docs/usage.rst:59]: > No # pylint:disable=invalid-name?

We don't use `pylint`, and `flake8` doesn't have problems with this, but we can add it for illustration in this case.

Comment (mrbean-bremen) [reply docs/usage.rst]: Actually you can ignore this comment - I overlooked that this in the documentation, not in the example code. I have added it as proposed.

**Pull request merged**